grants_tagger/tag_grants.py

```diff
@@ -27,17 +27,19 @@ def tag_grants(grant, threshold):
             ScienceCategory#{1..9}
             DiseaseCategory#{1..9)
     """
-    tags = predict_tags(grant['title'] + ' ' + grant['synopsis'], threshold=threshold)
-    tagged_grant = {
-        'Grant ID': grant['grant_id'],
-        'Reference': grant['reference'],
-        'Grant No.': grant['grant_no']
-    }
-    tagged_grant.update({
-        f"Science Category#{i+1}": tag
-        for i, tag in enumerate(tags)
-    })
-    return tagged_grant
+    grants_texts = [grant["title"]+["synopsis"] for g in grants]
```

Comment:
I am confused with this syntax. It looks like it won't work, because `g` wasn't defined. Should it be `for grant in grants]` ?

Reply:
Niice, well i did not write a test for this one, one more reminder why tests should be added for all code you introduce. Will add now.

Reply:
81abdda2c753ec4ec53e09680db9c8d028ce9b93